### PR TITLE
sign in user is created

### PR DESCRIPTION
**Pull request by abhishekhsheoran**

### CODE REVIEW

**Comment by abhishekhsheoran** [comment]
#10 
#9

File: users/users.go [hunk 1]
```diff
@@ -19,25 +39,45 @@ func CreateUsers(w http.ResponseWriter, r *http.Request) {
...
-		http.Error(w, "user id cannot be empty", http.StatusBadRequest)
+
+	verifyResult := verifyUserAttributes(user)
+	if verifyResult != "" {
+		http.Error(w, verifyResult, http.StatusBadRequest)
 		return
 	}
-	if user.Name == "" {
-		http.Error(w, "user name cannot be empty", http.StatusBadRequest)
+	collection := utils.GetCollection()
```
Comment (vpn-soft) [inline users/users.go]: Take collection name as input in this function

File: users/users.go [hunk 2]
```diff
@@ -8,8 +8,28 @@ import (
...
 	"github.com/abhishekhsheoran/library-management/utils"
+	"github.com/gorilla/mux"
+	"go.mongodb.org/mongo-driver/bson"
 )
 
+func verifyUserAttributes(user models.User) string {
+	var error string
+	if user.Name == "" {
+		error = "user name cannot be empty"
+		return error
```
Comment (vpn-soft) [inline users/users.go:19]: Make custom error here, the error returned is nil in this case. You can use fmt.Errorf() or errors.New() to create error with custom error message

**Review by vpn-soft — APPROVED** [review]
Lgtm

**Pull request merged**